Update README.md (#348)
mattermost/mattermost-handbook: operations/operations/mlt-cadence/README.md

* Chris Overton: VP, Engineering
* Lindsay Brock: Head of Product Management

> # MLT Cadance

Reviewer: ```suggestion
# MLT Cadence
```

## MLT Weekly Business Review (WBR)

**Tuesdays 10:00am to 11:00am Palo Alto time with MLT**

> **Required Attendees - MLT + Extended team (TBD)**

Reviewer: ```suggestion
**Required attendees: MLT + extended team (TBD)**
```

The purpose of MLT WBR is to review the health of the business by reviewing our goals dashboard, identifying key risks/challenges/issues, and identifying owners for actions. Discussions held in [MLT WBR Channel](https://community.mattermost.com/private-core/channels/MLT-WBR).

#### Pre-work

> Dashboard to be updated by all owners by Tuesday 9am Palo Alto Time.

Reviewer: ```suggestion
* Dashboard to be updated by all owners by Tuesday 9am Palo Alto Time.
```

> 2x2 prepared by each functional owner (TBD if reviewed separately or aggregated into one 2x2) (e.g., next product releases, marketing launches)

Reviewer: ```suggestion
* 2x2 prepared by each functional owner (TBD if reviewed separately or aggregated into one 2x2) (e.g., next product releases, marketing launches).
```

#### Process

> * **\(0:10\) Review WBR Agenda Doc** - Time for all to read data

Reviewer: ```suggestion
* **\(0:10\) Review WBR Agenda Doc:** Time for all to read data
```

> * **\(0:15\) MLT Good News -** Always start with anyone wanting to share the good news, e.g., customer wins, new release delivered, new web page launched, etc.

Reviewer: ```suggestion
* **\(0:15\) MLT Good News:**  Always start with anyone wanting to share the good news, e.g., customer wins, new release delivered, new web page launched, etc.
```

> * **\(0:20\) V2MOM Metrics Dashboard Review -** Review Dashboard

Reviewer: ```suggestion
* **\(0:20\) V2MOM Metrics Dashboard Review:** Review Dashboard
```

  * Discuss reds and new goals to create
  * Discuss yellows and paths to green
  * Review key updates to greens
  * Create new action items and assign to owners in Asana 

> * **\(0:25\) 2X2 Business Review -** Review WBR 2x2 document summarizing:  

Reviewer: ```suggestion
* **\(0:25\) 2X2 Business Review:** Review WBR 2x2 document summarizing:  
```

  * State of the business
    * Key business metrics (ARR, churn, customer count, health score)
    * Big updates from Sales, R&D, Marketing, Finance, etc.
  * Issues/Challenges/Risks
    * Key customer issues (at-risk customers, deals, etc.)
    * Key R&D risks/challenges/issues
    * Key marketing risks/challenges
  * Goals
  * Upcoming Milestones
  * Upcoming R&D milestones and dates
  * Upcoming marketing milestones and dates

> * **\(0:55\) Decisions/Actions -** Summarize decisions/actions in writing 

Reviewer: ```suggestion
* **\(0:55\) Decisions/Actions:** Summarize decisions/actions in writing 
```

## MLT Weekly Executive Meeting 

**Tuesdays 11:00am to 12:00pm Palo Alto time with MLT**

> **Required Attendees - MLT only + HR Representative**

Reviewer: ```suggestion
**Required attendees: MLT only + HR representative**
```

> The purpose of MLT Staff is to discuss MLT topics like CEO updates, board-level topics, hiring, sensitive HR-related announcements, company-wide programs like annual reviews & compensation, staff top of mind, etc.

Reviewer: ```suggestion
The purpose of the MLT Weekly Executive Meeting is to discuss MLT topics like CEO updates, board-level topics, hiring, sensitive HR-related announcements, company-wide programs like annual reviews and compensation, staff top of mind, etc.
```

Ian to moderate through Asana.

## MLT Weekly Deep Dive Meeting

**Wednesdays 9:00am to 10:30am Palo Alto time with MLT**

> **Required Attendees - MLT only + extended team (as appropriate by topic)  **

Reviewer: ```suggestion
**Required attendees: MLT only + extended team (as appropriate by topic)**
```

> The purpose of this new MLT weekly deep dive series is to achieve a deeper understanding and clarity on functional areas and/or strategic topics through reviews, discussions, and decisions.

Reviewer: ```suggestion
The purpose of this new MLT Weekly Deep Dive series is to achieve a deeper understanding and clarity on functional areas and/or strategic topics through reviews, discussions, and decisions.
```

> Topics are queued in a [central spreadsheet](https://docs.google.com/spreadsheets/d/1d_kBiMSsrjYVWX6JUjzNLruq8yQciIdIz-Vtmy69ygo/edit#gid=0) and prioritized by the Vice-Chair. Examples of topics are Cloud MVP, DevOps command center launch, customer journey & packaging proposals, marketing OKRs, etc.

Reviewer: ```suggestion
Topics are queued in a [central spreadsheet](https://docs.google.com/spreadsheets/d/1d_kBiMSsrjYVWX6JUjzNLruq8yQciIdIz-Vtmy69ygo/edit#gid=0) and prioritized by the Vice-Chair. Examples of topics are Cloud MVP, DevOps Command Center launch, customer journey and packaging proposals, marketing OKRs, etc.
```

#### Pre-work

> * (Vice-Chair) Agenda drafted based on queued topics in a central google docs sheet

Reviewer: ```suggestion
* (Vice-Chair) Agenda drafted based on queued topics in a central Google Doc
```

* (Owner) Prepare narrative document for the meeting

#### Process

> * 2 slots of 45 minutes each; Typically 2 topics for 2 slots but bigger topics can occupy 2 slots.

Reviewer: ```suggestion
* Two slots of 45 minutes each. Typically two topics - one for each slot. However, bigger topics can occupy two slots
```

> * MLT reviews document for the first 15 minutes; discusses for the next 25 minutes to reach any decisions for clarity and closure.

Reviewer: ```suggestion
* MLT reviews document for the first 15 minutes; discusses for the next 25 minutes to reach any decisions for clarity and closure
```

> * (5m before end of slot) Decisions/actions are summarized

Reviewer: ```suggestion
* (Five minutes before end of slot) Decisions/actions are summarized
```

The following should happen within a 2 week period, with 1-2 iterations in each step:

1. CEO discusses company and department V2MOMs 1-1 with department heads to ensure alignment

>    * If applicable during the period, sales, CEO and finance set or adjust the revenue targets

Reviewer: ```suggestion
   * If applicable during the period, Sales, CEO, and Finance set or adjust the revenue targets
```

> **T-Minus 2 Weeks to announcement.**

Reviewer: ```suggestion
**T-Minus 2 Weeks to announcement**
```

* At Monthly Strategic Meeting:
  * WIP V2MOMs, quarterly plan and proposed orgs reviewed by MLT
  * MLT shares feedback on each V2MOM and quarterly plan
  * Cross-department dependencies documented
  * Draft agenda
    * Review of MLT Plan Process

>     * Company V2MOM & Q2 plan

Reviewer: ```suggestion
    * Company V2MOM and Q2 plan
```

>     * Sales V2MOM, Q2 plan and proposed org

Reviewer: ```suggestion
    * Sales V2MOM, Q2 plan, and proposed org
```

>     * R&D V2MOM, Q2 plan and proposed org

Reviewer: ```suggestion
    * R&D V2MOM, Q2 plan, and proposed org
```

>     * Marketing V2MOM, Q2 plan and proposed org

Reviewer: ```suggestion
    * Marketing V2MOM, Q2 plan, and proposed org
```

>     * Customer Success V2MOM, Q2 plan and proposed org

Reviewer: ```suggestion
    * Customer Success V2MOM, Q2 plan, and proposed org
```

>     * Finance V2MOM, Q2 plan and proposed org

Reviewer: ```suggestion
    * Finance V2MOM, Q2 plan, and proposed org
```

>     * People V2MOM, Q2 plan and proposed org

Reviewer: ```suggestion
    * People V2MOM, Q2 plan, and proposed org
```

* MLT socializes updates to high-level plan with their departments

> **Finalize Plan**

Reviewer: ```suggestion
**Finalize plan**
```

> **T-Minus 1 Week to announcement**

Reviewer: ```suggestion
**T-Minus 1 week to announcement**
```

* Company and V2MOMs updated given feedback
* Financial plan adjusted
* Headcount approvals published

**Plan Shared**

**Week of announcement**

* Company V2MOM shared at all-hands
* Department V2MOMs are posted
* MLT presents company plan to their departments

## Quarterly Business Review \(QBR\)

Each quarter we review process towards our V2MOMs and discuss achievements and opportunities.

> #### Quarterly Business Review

Reviewer: I think this may be extraneous as there is already a QBR heading (H2) above it.

Author: I agree. We can remove the duplication.

QBR is an exercise to increase output through concise, efficient review of quarterly goals for company and department in the context of V2MOMs.

> **Time and People**

Reviewer: ```suggestion
**Time and people**
```

> * Scheduled 1 week after quarter end and can be broken out into

Reviewer: ```suggestion
* Scheduled 1 week after quarter end and can be broken out into:
```

    * 90 mins for Sales QBR (30 mins for each region)   
    * 45 mins CS QBR

>     * 2 hours for MLT QBR. (30 mins each for Product, Engineering, Q&A & Marketing) 

Reviewer: ```suggestion
    * 2 hours for MLT QBR (30 mins each for Product, Engineering, G&A, and Marketing) 
```

> **Prior to Meeting**

Reviewer: ```suggestion
**Prior to meeting**
```

* Materials for QBR agenda are shared by Friday prior to the QBR meeting.

**Agenda**

> * \(CEO\) Reviews [MLT Cadence V2MOM](http://handbook.mattermost.com/leadership/leadership-meetings.html#MLT-Cadence-VPMOM) \(10 minutes\)

Reviewer: ```suggestion
* \(CEO\) Reviews [MLT Cadence V2MOM](https://handbook.mattermost.com/leadership/leadership-meetings.html#MLT-Cadence-VPMOM) \(10 minutes\)
```

> **Q3 Punchlist for Next Fiscal Year Planning**

Reviewer: ```suggestion
**Q3 Punchlist for next fiscal year planning**
```

1. The company 3-year aspirations are reviewed
2. A 1% MLT V2MOM for next fiscal year is drafted by CEO and reviewed with MLT department heads
3. Department heads draft 1% Department V2MOMs for next fiscal year and review with peer stakeholders and CEO
4. At Q3 Planning Offsite, MLT reviews 50% MLT V2MOM for next fiscal year developed from 1% draft
5. At Q3 Planning Offsite, department heads present 50% Department V2MOMs, reviewed by peer stakeholders and CEO
6. Action items are documented and developed around Obstacles to success of next fiscal year
7. 50% MLT and Department V2MOMs, plus Obstacles, are presented to department leaders

#### Q4 Planning for next fiscal year

The focus of Q4 is arriving at a 99% draft of the plan for next fiscal with alignment across MLT and departments, with key Obstacles mitigated, and headcount and budgets reviewed.

1. MLT and middle managers work through key Obstacles for next fiscal year
2. Finance works with MLT to draft financials plan for next fiscal year
3. CEO works with MLT to develop MLT V2MOM from 50% to 99%

> 4. Department heads work with their teams, peer stakeholders and CEO to develop Department V2MOM from 50% to 99%

Reviewer: ```suggestion
4. Department heads work with their teams, peer stakeholders, and CEO to develop Department V2MOM from 50% to 99%
```

5. 99% MLT V2MOM is reviewed by MLT and agreed
6. 99% Department V2MOMs are reviewed by MLT peers and CEO and agreed
7. Plan for next fiscal year is locked based on V2MOMs

#### MLT V2MOM Review Process

> V2MOMs, quarterly plans and proposed orgs are reviewed by MLT. Use the following order to review V2MOMs:

Reviewer: ```suggestion
V2MOMs, quarterly plans, and proposed orgs are reviewed by MLT. Use the following order to review V2MOMs:
```
